playbooks/backup.yaml

```diff
@@ -0,0 +1,42 @@
+# Playbook: Set up backups.
+---
+- name: Set up backups
+  hosts: libvirt_hosts
+  become: true
+  tasks:
+    - name: Install needed packages
+      ansible.builtin.package:
+        name: restic
+        state: present
+
+    - name: Set up backup script
+      ansible.builtin.copy:
+        src: files/backup/backup.sh
+        dest: /usr/local/sbin/backup.sh
+        owner: root
+        group: root
+        mode: '0700'
+
+    - name: Save restic password
+      ansible.builtin.copy:
+        content: '{{ backup__restic_password }}'
+        dest: /usr/local/etc/restic_password
+        owner: root
+        group: root
+        mode: '0600'
+      no_log: true
+      when: backup__restic_password is defined
+
+    - name: Set up cron job
+      ansible.builtin.cron:
```

Comment:
At first this looked fine, but then I looked at the documentation of the `cron` module, and I think we should avoid it! It primarily manages per-user cron tabs (as opposed to system cron tabs stored in /etc/cron*), and has some really weird ideas on how to do things...

Instead, I'd suggest just using `copy` or `template` to place a file in `/etc/cron.daily`. You could also use `/etc/cron.d`, but I am not sure we need to customise the time of the backup..

Reply:
I done it via https://github.com/Tina-pm/ansible-playbooks/pull/20/commits/666a58dc739e6f8202f75ad934e854f0ec4c544c